packaging_automation/common_tool_methods.py

```diff
@@ -290,3 +323,171 @@ def remove_cloned_code(exec_path: str):
         except:
             print(f"Some files could not be deleted in directory {exec_path}. "
                   f"Please delete them manually or they will be deleted before next execution")
+
+
+def process_template_file(project_version: str, templates_path: str, template_file_path: str):
+    '''Process template files and return the processed template '''
```

Comment:
Can we put some comments to clarify a few things here ? Like:

* What is a template file and for which purposes we use them ?
* What does "process" mean here ? What do we do by calling "render" function ?

Reply:
Added some more comments. I hope this one is enough